src/dashboard/Data/Browser/Browser.react.js

```diff
@@ -320,9 +321,21 @@ export default class Browser extends DashboardView {
     return data;
   }
 
+  async fetchParseDataCount(source, filters) {
+    const query = queryFromFilters(source, filters);
+    const count = await query.count({ useMasterKey: true });
```

Comment:
@steven-supersolid I want to release a new version including the Jobs fix, so I'm ok merging this and improving later.